pyvista/plotting/renderer.py

```diff
@@ -1267,6 +1273,17 @@ def show_bounds(
             raise ValueError(f'padding ({padding}) not understood. Must be float between 0 and 1')
         cube_axes_actor.SetBounds(bounds)
 
+        # update the values of range of each axis
+        if isinstance(bounds_range, (list, tuple)):
```

Comment:
I'd consider being more liberal here. `collections.abc.Sequence` for one, but that doesn't include numpy ndarrays which might also appear here.

Reply:
There are quite a few occurrences of `isinstance(..., (list, tuple))` we may want to address

Reply:
@banesullivan that should probably be a separate PR.

Reply:
@adeak I agree. I can imagine the `isinstance()`growing quite large though. 

Any thoughts on an approach that can elegantly deal with this other than just adding to the tuple?

```python
if isinstance(..., (list, tuple, 
                    np.ndarray, collections.abc.Sequence)):
```

Reply:
`collections.abc.Sequence` includes stdlib sequences, so lists and tuples too.